portfolio/src/main/webapp/script.js

```diff
@@ -10,4 +10,16 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+const navigate = (pageid) => document.getElementById(pageid).scrollIntoView({behavior:'smooth', block:'start'});
```

Comment:
rename `pageid` to `pageId`, using [`lowerCamelCase`](https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined)

Reply:
Updated in commit 9e98000

Reply:
Updated in commit 9e98000